docs/en/vrf.md

```diff
@@ -1,42 +1,49 @@
 # WINkLink Random number Service
 
 ## Overview
-A Verifiable Random Function (VRF) is the public-key version of a keyed cryptographic hash. Only the holder of the private key can compute the hash, but anyone with public key can verify the correctness of the hash.
-VRF can be used to generate safe and reliable random numbers.
-Random number is determined by seed (provided by users), nonce (private state of VRFCoordinator contract) , hash of block and private key of random number generating node. Random number node cannot cheat. To keep the security, the random number is verified before it is sent to the user DAPP.
+Verifiable Random Function (VRF) is the public-key version of a keyed cryptographic hash, which can be used as random number. Only the holder of the private key can compute the hash, but anyone with public key can verify the correctness of the hash.
```

Comment:
Verifiable Random Function (VRF) is the public-key version of a keyed cryptographic hash, which can be used as a random number. Only the holder of the private key can compute the hash, but anyone with the public key can verify the correctness of the hash.
VRF can be used to generate secure and reliable random numbers.